liquibase-couchbase/src/test/java/system/change/InsertDocumentsSystemTest.java

```diff
@@ -0,0 +1,87 @@
+package system.change;
+
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.Scope;
+import common.operators.TestCollectionOperator;
+import liquibase.Liquibase;
+import liquibase.exception.LiquibaseException;
+import liquibase.ext.couchbase.types.DataType;
+import liquibase.ext.couchbase.types.Document;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import system.LiquibaseSystemTest;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static common.constants.ChangeLogSampleFilePaths.INSERT_DOCUMENTS_FAILED_TRANSACTION_TEST_XML;
+import static common.constants.ChangeLogSampleFilePaths.INSERT_DOCUMENTS_TEST_XML;
+import static common.constants.TestConstants.TEST_BUCKET;
+import static common.constants.TestConstants.TEST_COLLECTION;
+import static common.constants.TestConstants.TEST_SCOPE;
+import static common.matchers.CouchbaseCollectionAssert.assertThat;
+import static liquibase.ext.couchbase.types.Document.document;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+
+public class InsertDocumentsSystemTest extends LiquibaseSystemTest {
+    private static final Collection collection = bucketOperator.getCollection(TEST_COLLECTION, TEST_SCOPE);
+    private static final Scope scope = cluster.bucket(collection.bucketName()).scope(collection.scopeName());
+    private static final TestCollectionOperator collectionOperator = new TestCollectionOperator(collection);
+
+    @BeforeAll
+    static void setUp() {
+        createPrimaryIndex();
+    }
+
+    @AfterAll
+    static void tearDown() {
+        dropPrimaryIndex();
+    }
+
+    private static void createPrimaryIndex() {
+        Collection col = clusterOperator.getBucketOperator(TEST_BUCKET).
```

Comment:
This part looks like getting the same collection as the one defined in the class constants above, isn't it?

Reply:
Yes you are right, thank you